publications.rkt

```diff
@@ -51,9 +51,21 @@
 @(define ARRAY "Workshop on Libraries, Languages, and Compilers for Array Programming (ARRAY)")
 @(define ESECFSE "European Software Engineering Conference and Symposium on the Foundations of Software Engineering (ESEC/FSE)")
 @(define CSUR "ACM Computing Surveys (CSUR)")
+@(define JAR "Journal of Automated Reasoning (JAR)")
+@(define TECS "ACM Transactions on Embedded Computing Systems (TECS)")
 
 @(define publications
    (list
+    (publication "Verifying a Concurrent Garbage Collector with a Rely-Guarantee Methodology"
+                 "Yannick Zakowski, David Cachera, Delphine Demange, Gustavo Petri, David Pichardie, Suresh Jagannathan, and Jan Vitek"
+                 TECS
+                 2019
+                 "http://janvitek.org/pubs/tecs18.pdf")
+    (publication "How to Evaluate the Performance of Gradual Type Systems"
+                 "Ben Greenman, Asumu Takikawa, Max S. New, Daniel Felty, Robert Bruce Findler, Jan Vitek, and Matthias Felleisen"
```

Comment:
`Felty` is misspelled --- please change to `Feltey`

Reply:
Many thanks for the review! Is it ok to fix the typo in `master` directly? I have run into this kind of issues before and have not found elegant way to augment a merged PR.

Reply:
Yeah absolutely. Add a new commit.

Don't try to change the history of commits already pushed to master.

Reply:
Done with b9f39f1bc2.

Reply:
And thanks again!

@janvitek This was copied from your homepage, so you might want to fix the typo in Feltey's last name (JFP'18) there too. Btw if you had the homepage repo hosted somewhere, I could even PR it.